en/config/battery.md

```diff
@@ -0,0 +1,81 @@
+
+## Without load compensation
+With the default parameter set there is no load compensation enabled because if you don't have a current sensor, it might be uncalibrated or the internal resistance of your battery is completely off the resulting compensation might not make sense.
+
+If you don't want to use load compensation set the empty cell voltage to the value that should result in 0% battery estimation.
+
+## With load compensation
+If you use load compensation the empty voltage should be set higher than without because the compensated voltage gets used for the estimation. What you want to set is typically a bit below the expected rest cell voltage when empty after use.
+
+### With thrust based load compensation (not recommended)
+Without a current sensor it's very hard to do load compensation. You can enable compensation based on the total thrust which gets commanded to the motors by setting the parameter BAT_V_LOAD_DROP to how much voltage drop a cell shows under the load of full throttle. But this method is not accurate even with a good parameter because there's a delay between thrust command and current and the thrust in not linearly proportional to the current.
+
+### With current based load compensation (recommended)
+If you have a current sensor on board you can enable load compensation using the current measurement by setting the parameter `BAT_R_INTERNAL` to the internal resistance of your battery. There are li-po chargers out there which can measure the internal resistance of your battery. A typical value is 5mΩ but this can vary with discharge current rating, age and health of the cells.
+With decent configured load compensation the voltage used for the battery fuel estimation is much more stable and will not vary that much anymore when flying up and down.
+
+### Calibration Voltage Devider & Current per volt
+If you have a vehicle which measures voltage and current through a power module and the ADC of the FCU then you should check and calibrate the measurements once per board. To calibrate you'll need a multimeter.
+
+The easiest way to calibrate the dividers is by using QGroundControl and following the step-by-step guide on https://docs.qgroundcontrol.com/en/SetupView/Power.html.
+
+## Combning voltage based estimation with current integration
+After tuning the voltage based estimation you can achieve a more stable linear and accurate estimation by fusing your estimate with the so far used capacity. This is only possible with accurate current measurements because the current gets mathematically integrated over time to have a used capacity estimate.
+This is the best way to have totally accurate relative fuel consumption of the battery. You can not use only this technique because:
```

Comment:
I do not know what you mean by "You can not use only this technique because:"

I think what you are saying is that to use the technique you have to know the current battery capacity. Normally that means you would need to start with a full battery.

Reply:
The best way to track the used energy is to measure the current and accumulate it over time. But we want to know how much energy is left in the battery and not how much we used. So we need to know how much energy was available from the battery when we started counting.

We estimate how full it was in the beginning using the voltage estimate and assume the battery has in total the capacity you configured in the parameter. If one of those assumptions is wrong the battery might last longer or be empty earlier than we think. The first leads to less flight time and the second to a crash. That's why we still need to fuse the voltage estimate all the time especially when the battery is nearly empty. And also why the voltage estimate should be configured usefully first/for sure.

As smart battery usually always follows the state of the battery even if your device is turned off, the battery is removed, the battery is charged, degrades with cycles or gets damaged. So it basically has all data inside for a really accurate estimation of how much energy is really left in the battery.